packages/gasket-core/test/index.test.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-process-env */
+
+import { jest, expect } from '@jest/globals';
+import GasketEngine from '@gasket/engine';
+import { makeGasket } from '../lib/index.js';
+
+// eslint-disable-next-line no-unused-vars
+const warnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
+const errorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+
+/** @type {import('@gasket/engine').Plugin} */
```

Comment:
The tsconfig.json is currently configured to skip typechecking anything in the test directory.

Reply:
Ah yes good to know - I don't think it hurts to have these here for reference at least. Perhaps we should enable type checks for some tests?